contracts/Ethereum_SpokePool.sol

```diff
@@ -0,0 +1,70 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "./interfaces/WETH9.sol";
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import "./SpokePool.sol";
+import "./SpokePoolInterface.sol";
+
+/**
+ * @notice Ethereum L1 specific SpokePool.
+ * @dev Used on Ethereum L1 to facilitate L2->L1 transfers.
+ */
+
+contract Ethereum_SpokePool is SpokePoolInterface, SpokePool, Ownable {
```

Comment:
As discussed IRL, we might want to add a TODO to refactor the SpokePool inheritance setup s.t. we can have most of these methods implemented in the base contract and just have an `_calledByAdmin()` function that's implemented in each derived class, and have the base class call `require(_calledByAdmin(), "not authorized");`